### ISSUE TITLE
Cannot create a table with the name "order"

### ISSUE THREAD

**Issue by stefanosandes** [oguimbal/pg-mem#171]
# Describe the bug

pg-men cannot create a table named as `order`.

```
    create table ccca.order
                        ^
    Unexpected kw_order token: "order". Instead, I was expecting to see one of the following:

        - A "kw_primary" token
        - A "kw_unique" token
        - A "word" token
```

# To Reproduce

```sql
create schema ccca;
create table ccca.order (
	id serial,
	coupon text,
	code text,
	cpf text,
	issue_date timestamp,
	freight numeric,
	total numeric,
	sequence integer,
	primary key (id)
);
```

# pg-mem version

2.1.9

**Comment by oguimbal** [comment]
Will fix.

Woraround: quote the name (`"order"` instead of `order`)

**Comment by oguimbal** [comment]
Hi ! Fixed with `pg-mem@2.3.0`
